portfolio/src/main/webapp/task.js

```diff
@@ -0,0 +1,8 @@
+class Task{
+  constructor(name, description, durationMinute, priority) {
```

Comment:
durationMinute => durationMinutes. Also, the comment should indicate which fields are optional. And it should indicate how the optional status is conveyed (null? undefined?)

Reply:
Done.